### PR TITLE
HomePage

### PR DESCRIPTION
**Pull request by shirishgoyal**
MD layout for homepage and SCSS

image

### CODE REVIEW

**Comment by dmorina** [comment]
we will return to these, atm low priority, thanks

File: static/css/_card.scss [hunk 1]
```diff
@@ -0,0 +1,5 @@
+@import 'variables';
+
+md-card{
+  outline:0;
+}
```
Comment (dmorina) [inline static/css/_card.scss]: not necessary to have its own scss file, originally it has one dedicated scss in angular-material

**Comment by dmorina** [comment]
resolve conflicts to continue with review

**Comment by shirishgoyal** [comment]
Made some changes as per discussion with Karolina
Updated Font to Source Sans Pro from Roboto
Removed Table Shadow

Resolved height issues in dashboard, My projects views
All code merge conflicts resolved

**Comment by aginzberg** [comment]
"How it works" link keeps you on login. I assume it should take you to the homepage?

Spacing is a bit funky on a smaller screen:
screen shot 2015-11-04 at 11 16 28 am

**Comment by aginzberg** [comment]
our profiles pictures should probably be fixed size

File: static/css/_button.scss [hunk 2]
```diff
@@ -0,0 +1,13 @@
+button{
```
Comment (dmorina) [inline static/css/_button.scss]: unless we are implementing new components we shouldn't have these scss files, it's a bit confusing, move the styles to base or elsewhere

File: static/css/app.css [hunk 3]
```diff
@@ -289,7 +327,7 @@ md-data-table {
   md-data-table .md-data-table-header-column.sorted {
     color: rgba(0, 0, 0, 0.87); }
   md-data-table .md-data-table-row-cell {
-    color: rgba(0, 0, 0, 0.87);
+    color: #243058;
```
Comment (dmorina) [inline static/css/app.css]: leave it black, considering the specs says it should be black, should be overriden somewhere else

File: static/templates/authentication/login.html [hunk 4]
```diff
@@ -1,3 +1,12 @@
+<div>
+    <div flex layout="row" layout-align="center start">
+        <h1 class="md-display-3" layout="column" layout-align="center"><img src="/static/images/logo.svg"> DAEMO</h1>
+    </div>
+    <div flex layout="row" layout-align="center">
+        <md-button class="md-raised md-primary" ng-href="/login">Sign In</md-button>
```
Comment (dmorina) [inline static/templates/authentication/login.html]: remove signin button on top if the form is active

File: static/templates/authentication/register.html [hunk 5]
```diff
@@ -1,3 +1,12 @@
+<div>
+    <div flex layout="row" layout-align="center start">
+        <h1 class="md-display-3" layout="column" layout-align="center"><img src="/static/images/logo.svg"> DAEMO</h1>
+    </div>
+    <div flex layout="row" layout-align="center">
+        <md-button class="md-raised md-primary" ng-href="/login">Sign In</md-button>
+        <md-button class="md-raised md-accent"  ng-href="/register">Register</md-button>
+    </div>
```
Comment (dmorina) [inline static/templates/authentication/register.html]: remove sign-up button if we are on /register

**Comment by shirishgoyal** [comment]
Removed "Our Profiles" button for now.
Handled all changes as per feedback.

**Comment by dmorina** [comment]
we will redesign this based on new Master design 
reOpen if it becomes relevant again

**Pull request closed without merging**